src/main/java/se/sundsvall/emailsender/service/MicrosoftGraphMailSender.java

```diff
@@ -83,8 +89,9 @@ public void sendEmail(final SendEmailRequest request) {
 				.post(requestBody);
 		} catch (final Exception e) {
 			throw Problem.builder()
```

Comment:
Why not just throw an ordinary Exception (and setting the cause)?
The Exception will be wrapped into a Problem higher up in the chain, right?

Reply:
`throw new IOException("Unable to send e-mail");`

?